compaction/task_manager_module.cc

```diff
@@ -281,14 +282,38 @@ sstring major_compaction_task_impl::to_string(flush_mode fm) {
     __builtin_unreachable();
 }
 
+static future<bool> maybe_flush_all_tables(sharded<replica::database>& db) {
+    auto interval = db.local().get_config().compaction_flush_all_tables_before_major_seconds();
+    if (interval) {
+        auto when = db_clock::now() - std::chrono::duration_cast<db_clock::duration>(std::chrono::seconds(interval));
```

Comment:
I'm guessing you can write `auto when = db_clock::now() - interval * 1s;`

Reply:
done in v6